server/pbench/bin/pbench-dispatch

```diff
@@ -76,7 +76,7 @@ else
 
     # get the list of files we'll be operating on - sort them by size
     list=$TMP/$PROG.$$
-    find -L $ARCHIVE/*/$linksrc -name '*.tar.xz' 2>/dev/null | grep -v DUPLICATE > $list
+    find -L $ARCHIVE/hostnames/*/$linksrc -name '*.tar.xz' 2>/dev/null | grep -v DUPLICATE > $list
```

Comment:
Leave the old scripts alone: they do *NOT* change. It's only the new scripts that need to worry about the new structure.